client/src/components/Notifications/Broadcasts/BroadcastContainer.vue

```diff
@@ -31,125 +37,260 @@ const props = defineProps<{
-            </BButton>
-
-            <div v-if="remainingBroadcastsCountText" class="text-center mt-2">
-                {{ remainingBroadcastsCountText }}...
+
+                <div v-if="multiple" class="page-indicator">{{ currentPage + 1 }} / {{ sortedBroadcasts.length }}</div>
             </div>
-        </BCol>
-    </BRow>
+        </section>
+
+        <BButton v-if="multiple" class="arrow right inline-icon-button area-r" title="Next" @click="currentPage += 1">
+            <FontAwesomeIcon fixed-width icon="fa-chevron-right" />
+        </BButton>
+
+        <BButton class="dismiss-button inline-icon-button area-x" title="Dismiss" @click="dismiss">
+            <FontAwesomeIcon fixed-width icon="fa-times" />
+        </BButton>
+    </div>
 </template>
 
 <style lang="scss" scoped>
-.broadcast-banner {
-    width: 100%;
-    color: white;
-    display: flex;
-    z-index: 9999;
-    padding: 1rem;
-    min-height: 6rem;
-    backdrop-filter: blur(0.2rem);
-    justify-content: space-between;
-    background-color: rgb(0, 0, 0, 0.7);
-    box-shadow: 0 0 1rem 0 rgba(0, 0, 0, 0.5);
-
-    .broadcast-message {
-        font-size: large;
-    }
-}
+@import "theme/blue.scss";
 
-.fixed-position {
+$margin: 1rem;
+
+.broadcast-container {
     position: fixed;
-}
+    bottom: $margin;
+    left: 50%;
+    transform: translate(-50%, 0);
+    height: 200px;
+    width: min(calc(100% - $margin - $margin), 1200px);
+    background: $white;
+    border-color: $border-color;
+    border-width: 1px;
+    border-style: solid;
+    border-radius: 0.5rem;
+    display: grid;
+    grid-template-columns: 50px 1fr 50px;
+    grid-template-rows: 50px 1fr 50px;
+    grid-template-areas:
+        "i m x"
+        "l m r"
+        ". m .";
+
+    .area-l {
+        grid-area: l;
+    }
+    .area-m {
+        grid-area: m;
+    }
+    .area-x {
+        grid-area: x;
+    }
+    .area-r {
+        grid-area: r;
+    }
+    .area-i {
+        grid-area: i;
+    }
+
+    &.preview {
+        position: relative;
+        bottom: 0;
+    }
+
+    &.warning {
+        border-color: $brand-warning;
+    }
+
+    &.urgent {
+        border-color: $brand-info;
+        background: linear-gradient(to top, lighten($brand-info, 50%), $white 120px);
+    }
+
+    .arrow {
+        font-size: 2rem;
+
+        &.left {
+            border-top-left-radius: 0;
+            border-bottom-left-radius: 0;
+        }
+
+        &.right {
+            border-top-right-radius: 0;
+            border-bottom-right-radius: 0;
+        }
+    }
+
+    .dismiss-button {
+        font-size: 1.5rem;
+        color: $border-color;
+
+        &:hover {
+            background: none;
+            color: $brand-danger;
+        }
+
+        &:focus {
+            color: $brand-primary;
+        }
+    }
+
+    .info-icon {
+        display: grid;
+        place-items: center;
+        font-size: 1.5rem;
+
+        .urgent {
+            color: $brand-info;
+        }
```

Comment:
The `urgent` variant needs to stand out more than the rest. Probably using a "danger" like color as @martenson suggested will work. The use case for those types of broadcasts could be notifying issues about temporal degraded server performance or unplanned imminent emergency maintenance, etc.